docs/docs/10.4-test-utils.md

```diff
@@ -16,15 +16,22 @@ Simulate.{eventName}(DOMElement element, object eventData)
 
 Simulate an event dispatch on a DOM node with optional `eventData` event data. **This is possibly the single most useful utility in `ReactTestUtils`.**
 
-Example usage:
+**Clicking an element**  
+```javascript
+var node = React.findDOMNode(this.refs.button);
+React.addons.TestUtils.Simulate.click(node);
+```
 
+**Changing the value of an input field and then pressing ENTER**  
 ```javascript
 var node = React.findDOMNode(this.refs.input);
-React.addons.TestUtils.Simulate.click(node);
-React.addons.TestUtils.Simulate.change(node, {target: {value: 'Hello, world'}});
-React.addons.TestUtils.Simulate.keyDown(node, {key: "Enter"});
+node.value = 'giraffe'
```

Comment:
@zpao I'm fine with whatever you guys ultimately decide.  For a point of reference, this was one thing that tripped me up for a few minutes the first time I ran across it (which was pretty early in my usage of Simulate).  I suspect that this is a pretty common stumbling point among users of Simulate (it is the most obvious thing to do inside an event handler).  I don't feel strongly either way though.